beaver-builder/basics/multi-layer-backgrounds.md

```diff
@@ -0,0 +1,142 @@
+
+<details><summary>Adding Gradient Colors</summary>
+
+When you open the gradient color picker, it includes two preset color stop items by default, giving you a basic gradient to work with. To change a color in the gradient:
+
+1. Click on one of the color stop items in the gradient range control (the bar showing the gradient).
+2. The color swtch will update to show the color of the selected color stop.
+3. Click the color swatch to open the color picker.
+4. Choose your preferred color from the picker. Once selected, the gradient updates automatically.
+5. Repeat these steps to change the other color in the gradient.
+
+</details>
+
+<details><summary>Adding More Colors</summary>
+
+You can add more color stops to the gradient:
+
+1. Click anywhere on the gradient range control where there isn’t already a color stop item.
+2. A new color stop item will appear at that spot.
+3. Click the new color stop item, then the color swatch, and use the color picker to choose a color for it.
+
+</details>
+
+<details><summary>Adjusting the Gradient</summary>
+
+You can drag the color stop items left or right to adjust the position of each color in the gradient.
+
+</details>
+
+<details><summary>Delete a Gradient Item</summary>
+
+If you want to remove a color from the gradient:
+
+1. Click the gradient color swatch item you want to delete.
+2. Click the **Trash** button to the right-hand side of the color swatch.
+
+</details>
+
+<details><summary>Distrubte Gradient Items</summary>
+
+If your gradient has multiple color stops, you can easily distribute them evenly across the gradient. Simply click the Distribute button, and all color stops will be spaced out evenly.
+
+This feature is especially helpful when working with several color stops and aiming for a balanced look. After distributing, you can still fine-tune the colors and positions to achieve your desired gradient effect.
+
+</details>
+
+### Photo
+
+The Photo background type applies an image to the entire row, column, or Box module. You can choose images from the WordPress media library, a URL, or an SVG using SVG code.
+
+The Photo background type offers the following options:
+
+<details><summary>WordPress Media Library</summary>
+
+- **Select Image** - The Select Image button allows you to choose an image from the WordPress media library.
+- **WordPress Image Size** - Choose the WordPress media image size to use. You can choose from the default thumbnail, medium, large, or full size and any custom image size you have added.
+- **Replace** - Allows you to select a different image from the WordPress Media Library to replace the current background image.
+- **Remove** - Remove the image from the background.
+
+</details>
+
+<details><summary>URL</summary>
+
+- **URL** - Enter or paste the URL of the image you want to use into the URL field.
+
+</details>
+
+<details><summary>SVG</summary>
```

Comment:
I would suggest a note that the simpler the better for the SVG to work properly or something of that nature.